src/KafkaSchemaRegistryApiApiClient.php

```diff
@@ -0,0 +1,229 @@
+<?php
+
+namespace Jobcloud\KafkaSchemaRegistryClient;
+
+use Buzz\Exception\RequestException;
+use Exception;
+use Jobcloud\KafkaSchemaRegistryClient\Exception\SchemaNotFoundException;
+use Jobcloud\KafkaSchemaRegistryClient\Exception\SubjectNotFoundException;
+
+class KafkaSchemaRegistryApiApiClient implements KafkaSchemaRegistryApiClientInterface
+{
+    /**
+     * @var HttpClientInterface
+     */
+    private $httpClient;
+
+    /**
+     * KafkaSchemaRegistryApi constructor.
+     * @param HttpClientInterface $httpClient
+     */
+    public function __construct(HttpClientInterface $httpClient)
+    {
+        $this->httpClient = $httpClient;
+    }
+
+    /**
+     * @return array
+     */
+    public function getSubjects(): array
+    {
+        return $this->httpClient->call('GET', 'subjects') ?? [];
+    }
+
+    /**
+     * @param string $subjectName
+     * @return array
+     */
+    public function getAllSubjectVersions(string $subjectName): array
+    {
+        return $this->httpClient->call('GET', sprintf('subjects/%s/versions', $subjectName)) ?? [];
+    }
+
+    /**
+     * @param string $subjectName
+     * @param string $version
+     * @return string
+     */
+    public function getSchemaByVersion(string $subjectName, string $version = self::VERSION_LATEST): string
+    {
+        return $this
+                ->httpClient
+                ->call('GET', sprintf('subjects/%s/versions/%s', $subjectName, $version))['schema'];
```

Comment:
```suggestion
                ->call('GET', sprintf('subjects/%s/versions/%s', $subjectName, $version));
```
this is why in my previous review i asked you to add the call that does this :smile: , please check my old comment:
https://github.com/jobcloud/php-kafka-schema-registry-client/pull/1#discussion_r353627460

Reply:
added